documentation/modules/adding-source-provider.adoc

```diff
@@ -4,9 +4,14 @@
 
 [id="adding-source-provider_{context}"]
 ifdef::vmware[]
-= Adding a VMware source provider
+= Adding a VMware vSphere source provider
 
-You can add a VMware source provider by using the {ocp} web console.
+You can add a VMware vSphere source provider by using the {ocp} web console.
+
+[IMPORTANT]
+====
+EMS enforcement is disabled for migrations with VMware vSphere source providers because vSphere does not comply with the 2023 FIPS requirements. Therefore, users should consider whether migrations from vSphere source provider risk their compliance with FIPS.
```

Comment:
I think we should soften the wording here, how about
```suggestion
EMS enforcement is disabled for migrations with VMware vSphere source providers to enabling migrations from supported versions of vSphere (see ...) that do not comply with the 2023 FIPS requirements. Therefore, users should consider whether migrations from vSphere source provider risk their compliance with FIPS.
```

Reply:
Done.